src/alpaca_compiled_po.erl

```diff
@@ -0,0 +1,111 @@
+%%EPO COMPILED FILE
```

Comment:
Not any longer, since we automatically generate it. I've removed it and added it to .gitignore.